### DISCUSSION TITLE
ZeroTier and ActivityWatch

### DISCUSSION THREAD

**Discussion by edugrt** [ActivityWatch/activitywatch#743] [Q&A] [answered]
Hey!

ActitivyWatch is a great solution, thanks for all the effort!

Has anyone already tried to use AW through ZeroTier? I have two workstations, on different locations, both being able to communicate with each other through ZeroTier. One of them is running the AW Server and both are running AW Watchers. Just changing the IP address on the remote location didn't do the trick. Am I missing something?

Thanks

**Comment by ErikBjare** [comment, marked as answer]
Hi there! 

I've never heard about ZeroTier before, but it looks awesome (I've wanted something like it for years) so I just set it up on 4 of my machines. Very happy with how it works, thank you so much for the tip! (I even tweeted about it: https://twitter.com/ErikBjare/status/1532672577621073920)

>Just changing the IP address on the remote location didn't do the trick. Am I missing something?

What do you mean "changing the IP address on the remote"?

You need to poke in several config files. Some for the server (to bind on all/non-local network interface, and to allow viewing the web UI from a non-local origin), and some for the watchers (to report to the right server). Pretty sure that's it.

I think this has been answered a few times before, but I added it to the docs so now there's a somewhat canonical answer: https://docs.activitywatch.net/en/latest/remote-server.html#opening-the-server-to-the-network

If you're having trouble getting it to work, just let me know!

**Reply by xeruf** [reply]
Where is the setting to expose the Web UI? I have set the options in the server as outlined in the docs but still get a Connection Refused :/
